shared/login/relogin/index.native.js

```diff
@@ -8,6 +8,21 @@ import {isDeviceSecureAndroid, isAndroidNewerThanM, isAndroid} from '../../const
 import type {Props} from '.'
 
 class LoginRender extends Component<Props> {
+  _inputRef = React.createRef<Kb.Input>()
+
+  _focusInput = () => {
+    if (this._inputRef.current) {
+      this._inputRef.current.focus()
+    }
+  }
+
+  _selectedUserChange = selectedUser => {
+    this.props.selectedUserChange(selectedUser)
+    // For some reason, calling this immediately doesn't work, at
+    // least on iOS.
+    setImmediate(this._focusInput)
```

Comment:
i had to do this elsewhere recently also . i wonder if we want to just add this to common-adapters/input

Reply:
Sure. Where else did you have to add this? I can then fix it up when I add it to common-adapters/input.